lib/swagger/blocks.rb

```diff
@@ -248,6 +249,10 @@ def data
         @data ||= {}
       end
 
+      def keys(values)
```

Comment:
"values" sounds like a list, but it is in fact a dictionary. Please rename to "data"